app/common/lib/ledgerUtil.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+/**
+ * Returns true if HTTP response code is one we want to collect usage for
+ * @param {number} responseCode - HTTP response code to be evaluated
+ * @return {boolean} true if the page should be excluded, false if not
+ */
+module.exports.IsValidResponseCode = (responseCode) => {
```

Comment:
actually I take that back. We need to further restrict 2xx to 200, 203 and 206 I think